src/rest_service/result/models.py

```diff
@@ -0,0 +1,37 @@
+from django.db import models
+from django.utils.translation import gettext_lazy as _
+import uuid
+
+# Create your models here.
+
+class Result(models.Model):
+    id = models.UUIDField(
+        _("Id"),
+        primary_key=True,
+        db_column="result_id",
+        db_index=True,
+        auto_created=True,
+        unique=True,
+        editable=False,
+        default=uuid.uuid4
+    )
+
+    # In team_id ForeignKey cant write 'team.Team.id'
+    # is 'team.Team' ok?
+    # This goes for scenario_id too..
+    team_id = models.ForeignKey('team.Team', on_delete=models.CASCADE)  # should on_delete stayed on CASCADE or change?
+    scenario_id = models.ForeignKey('test.Scenario', on_delete=models.CASCADE) # should on_delete stayed on CASCADE or change?
+    
+    avarage_time = models.IntegerField() # i think it need to change !!
```

Comment:
yes i think it should be float
```suggestion
    avarage_time = models.FloatField()
```